Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Types/FarePolicy/FarePolicyRentalDetails.hs

```diff
@@ -23,8 +23,8 @@ data FPRentalDetailsD (s :: UsageSafety) = FPRentalDetails
   { baseFare :: Money,
     perHourCharge :: Money,
     distanceBuffers :: NonEmpty (FPRentalDetailsDistanceBuffersD s),
-    perHourFreeKms :: Int,
     perExtraKmRate :: Money,
+    perExtraMinRate :: Money,
```

Comment:
What about planned extra KMs?

Reply:
That logic will come from frontend I suppose? We can just store estimated fare and use it